community/ndp/kernelextension/pom.xml

```diff
@@ -5,10 +5,10 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>org.neo4j</groupId>
-    <artifactId>parent</artifactId>
+    <groupId>org.neo4j.ndp</groupId>
+    <artifactId>neo4j-data-protocol-parent</artifactId>
```

Comment:
This breaks from how we've done parents elsewhere - not sure if that's good or bad, but it's different :)

Other modules in the project use the `org.neo4j:parent` parent, that way there's a single parent for all the modules. See for instance: https://github.com/neo4j/neo4j/blob/2.3/community/function/pom.xml and note that that project is under the 'community' multi-module.

I think that the philosophy has been that the multi-module projects organize the modules in a tree, but they all inherit the same parent for ease of configuration.